amanuensis/resources/institution/__init__.py

```diff
@@ -3,4 +3,32 @@
 	get_all_institutions,
     update_institution,
     add_institution,
-)
\ No newline at end of file
+)
+import requests, json
+def api_request(name):
+    """
+    Makes a call to the Consolidated Screening List api of developer.trade.gov. Information returned in the dictionary
+    can be accessed by info_dict["results"], which is a list of dictionaries containing information about the legality
+    of interacting with the returned companies.
+    """
+    try:
+        url = f"https://data.trade.gov/consolidated_screening_list/v1/search?name={name}"
+        hdr ={
+        # Request headers
+        'Cache-Control': 'no-cache',
+        'subscription-key': 'a7c2cd5a313d430c9df8bc7918f2e14b',
```

Comment:
is this subscription key personal or public?

move this to the config file, if private do not commit it and share it via https://onetimesecret.com

Reply:
It is a personal subscription key.
The website generated this link which has the API key: https://onetimesecret.com/secret/4kdalq354mhqouo7ch9apzsg5kkjwp
It is now no longer in the code and has to be manually entered into the config file, is this correct?